babel.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = api => {
         targets: {
           esmodules: true,
         },
-        modules: false,
+        modules: test && 'auto',
```

Comment:
Is there a reason this is explicitly set to false for environments other than test?

I think it should always be `auto`, which is the same as omitting it.

Reply:
I need it to be off for UMD packaging, had some issue with babelify / browserify on this part as well... I think this might change when I am able to add karma-webpack with babel-loader.